iset.mm

```diff
@@ -60969,6 +61036,73 @@ the cancellation property (fifth hypothesis), show that the relation
+      cop cin opeldmg dmres eleq2i syl6ib elin simplbi syl6 mtod unsnfi syl3anc
+      wi eqeltrd syl5eqel ex findcard2sd eqeltrrd ) BAUAZAFGZHZBAIZBFWAWDBSWBAB
+      UBUCWCBCJZIZFGBKIZFGZBDJZIZFGZBWIEJZLZMZIZFGZWDFGCDEAWEKSWFWGFWEKBNOWEWIS
+      WFWJFWEWIBNOWEWNSWFWOFWEWNBNOWEASWFWDFWEABNOWHWCWGKFBUDUEUFUGWCWIFGZHZWIA
+      UHZWLAWIULZGZHZHZWKWPXCWKHZWOWJBWMIZMZFBWIWMUIXDXFWJWLWLBUJZVCZLZMZFXDXEX
+      IWJXDWAWLAGZXEXISWAWBWQXBWKUKZXDWLAWIWRWSXAWKUMZUNZAWLBUOPUPXDWKXHQGZXHWJ
+      GZUQXJFGXCWKRXDWLQGXGQGZXOXDWLWTXMURXDWAXKXQXLXNXQAWLBWLBUSUTPZWLXGQQVAPX
+      DXPWLWIGZXDWLAWIXMVBXDXPWLWIBTZVDZGZXSXDXPWLWJTZGZYBXDXKXQXPYDVOXNXRWLXGW
+      JAQVEPYCYAWLBWIVFVGVHYBXSWLXTGWLWIXTVIVJVKVLWJXHQVMVNVPVQVRWAWBRVSVT $.
+  $}
+
+  $( The domain of a finite function is finite.  (Contributed by Jim Kingdon,
+     5-Feb-2022.) $)
+  fundmfi $p |- ( ( A e. Fin /\ Fun A ) -> dom A e. Fin ) $=
+    ( cfn wcel wfun cdm cen wbr fundmeng enfii syldan ) ABCADAEZAFGKBCABHKAIJ
+    $.
+
+  $( A function (set) is finite if and only if its domain is finite.
+     (Contributed by AV, 10-Jan-2020.) $)
+  fundmfibi $p |- ( Fun F -> ( F e. Fin <-> dom F e. Fin ) ) $=
+    ( wfun cfn wcel cdm fundmfi ancoms wfn funfn fnfi sylanb impbida ) ABZACDZA
+    EZCDZNMPAFGMAOHPNAIOAJKL $.
+
+  $( If a set is finite, its converse is as well.  (Contributed by Jim Kingdon,
+     5-Feb-2022.) $)
+  relcnvfi $p |- ( ( Rel A /\ A e. Fin ) -> `' A e. Fin ) $=
+    ( wrel cfn wcel wa ccnv cen wbr wceq dfrel2 biimpi adantr simpr eqeltrd cvv
+    relcnv cnvexg cnven sylancr adantl enfii syl2anc ) ABZACDZEZAFZFZCDUFUGGHZU
+    FCDUEUGACUCUGAIZUDUCUIAJKLUCUDMNUDUHUCUDUFBUFODUHAPACQUFORSTUFUGUAUB $.
+
+  $( The range of a finite relation is finite if its converse is a function.
+     (Contributed by Jim Kingdon, 5-Feb-2022.) $)
+  funrnfi $p |- ( ( Rel A /\ Fun `' A /\ A e. Fin ) -> ran A e. Fin ) $=
+    ( wrel ccnv wfun cfn wcel w3a crn cdm df-rn relcnvfi 3adant2 simp2 syl5eqel
+    fundmfi syl2anc ) ABZACZDZAEFZGZAHRIZEAJUAREFZSUBEFQTUCSAKLQSTMROPN $.
+
+  $( A 1-1 function (class) with a set as domain is finite if and only if its
+     range is finite.  (Contributed by AV, 10-Jan-2020.) $)
+  f1dmvrnfibi $p |- ( ( A e. V /\ F : A -1-1-> B )
+                      -> ( F e. Fin <-> ran F e. Fin ) ) $=
+    ( wcel wf1 wa cfn crn wrel ccnv wfun f1rel ad2antlr wf1o f1cnv syl simpr wb
+    wceq f1ofun funrnfi syl3anc cdm cen wbr wi f1dm f1f1orn eleq1 f1oeq2 eqcoms
+    anbi12d biimpd expcomd sylc impcom adantr f1oeng enfii syl2anc f1fun mpbird
+    fundmfibi impbida ) ADEZABCFZGZCHEZCIZHEZVHVIGCJZCKZLZVIVKVGVLVFVIABCMNVGVN
+    VFVIVGVJAVMOVNABCPVJAVMUAQNVHVIRCUBUCVHVKGZVICUDZHEZVOVKVPVJUEUFZVQVHVKRVOV
+    PDEZVPVJCOZGZVRVHWAVKVGVFWAVGVPATZAVJCOZVFWAUGABCUHABCUIWBVFWCWAWBVFWCGZWAW
+    DWASAVPAVPTVFVSWCVTAVPDUJAVPVJCUKUMULUNUOUPUQURVPVJDCUSQVPVJUTVAVOCLZVIVQSV
+    GWEVFVKABCVBNCVDQVCVE $.
+
+  $( A 1-1 function (set) is finite if and only if its range is finite.
```

Comment:
> An injective function which is a set is finite if and only if its range is finite.  See also ~ f1dmvrnfibi .

? (ping @avekens )

Reply:
I will change this to "A one-to-one function which is a set is finite if and only if its range is finite. See also ~ f1dmvrnfibi ." in set.mm.

Reply:
I'll use this version of the comment.